Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -138,11 +138,27 @@ void DMainWindow::StartGame(const QString filename)
 	}
 	else
 	{
-		// TODO: Disable screensaver!
+		DisableScreensaver();
 		emit CoreStateChanged(Core::CORE_RUN);
 	}
 }
 
+void DMainWindow::DisableScreensaver()
+{
+#ifdef Q_OS_WIN
+	// TODO: Prevent display from sleeping. This only currently prevents the system from truely sleeping.
+	SetThreadExecutionState(ES_CONTINUOUS | ES_DISPLAY_REQUIRED);
```

Comment:
Unless I'm mistaken, DISPLAY_REQUIRED ensures the display is always on, and ES_SYSTEM_REQUIRED makes the system be always on?